packages/wallet-adapter-core/src/ans.ts

```diff
@@ -1,13 +1,16 @@
+import { AccountAddress } from "@aptos-labs/ts-sdk";
+
 const ChainIdToAnsContractAddressMap: Record<string, string> = {
   "1": "mainnet", // mainnet
   "2": "testnet", // testnet
 };
 
 export const getNameByAddress = async (
-  chainId: string,
-  address: string
+  chainId: string | number,
+  address: string | AccountAddress
```

Comment:
Nice, finally.

We might want to actually add get primary name by address as well as the main replacement for this (or use the V2 SDK directly)